cli/src/commands/get/comments.rs

```diff
@@ -328,6 +358,7 @@ pub fn get_many(client: &Client, args: &GetManyCommentsArgs) -> Result<()> {
         show_progress: !no_progress,
         label_attribute_filter,
         user_properties_filter,
+        messages_filter: Some(messages_filter),
```

Comment:
Should this be `None` if both senders and recipients are None? Or does it not really matter api/perf wise

Reply:
It doesn't matter api wise :)